ts/client/src/accounts/group.ts

```diff
@@ -472,11 +470,19 @@ export class Group {
       uiPrice = this.toUiPrice(price, baseDecimals);
       lastUpdatedSlot = stubOracle.lastUpdateSlot.toNumber();
       provider = OracleProvider.Stub;
+      deviation = stubOracle.deviation;
     } else if (isPythOracle(ai)) {
       const priceData = parsePriceData(ai.data);
       uiPrice = priceData.previousPrice;
       price = this.toNativePrice(uiPrice, baseDecimals);
       lastUpdatedSlot = parseInt(priceData.lastSlot.toString());
+      deviation =
+        priceData.confidence !== undefined
+          ? I80F48.fromNumber(
+              priceData.previousConfidence *
```

Comment:
since previousConfidence is always a number, can we replace all of this with `deviation = this.toNativePrice(priceData.previousConfidence, baseDecimals)` to parallel the `price`?